src/fake-timers-src.js

```diff
@@ -204,11 +205,17 @@ function withGlobal(_global) {
         _global.setImmediate = _global.setImmediate;
         _global.clearImmediate = _global.clearImmediate;
     }
+
+    if (intlPresent) {
+        _global.Intl = _global.Intl;
+    }
+
```

Comment:
Drop this "tautology". The above stuff is only a custom hack for IE, which does not have Intl anyway. I will remove the above, as we no longer support IE.